README.md

```diff
@@ -2,8 +2,8 @@
 
 > [!WARNING]
 > This action is deprecated and will be removed in a future release. 
-> Please use the `sonarqube-scan-action` action instead. 
-> The `sonarqube-scan-action` is a drop-in replacement for this action.
+> Please use the `official-sonarqube-scan` action instead. 
```

Comment:
```suggestion
> Please use the `sonarqube-scan-action` action instead. 
```
I would keep the name of the action as it was before (repo slug), because this is what users will have to type in their yaml pipeline.